.github/workflows/publish_plugin_to_hub.yaml

```diff
@@ -0,0 +1,120 @@
+      plugin_dir: ${{ steps.split.outputs.plugin_dir }}
+      prerelease: ${{ steps.semver_parser.outputs.prerelease }}
+      plugin_releaser: ${{ steps.set-result.outputs.result }}
+    steps:
+      - name: Split tag
+        id: split
+        run: |
+          tag=${{ github.ref_name }}
+          plugin_kind=source
+          plugin_name=cq-source-scaleway
+          plugin_kind_plural=sources
+          plugin_version=$tag
+          echo "plugin_name=${plugin_name}" >> $GITHUB_OUTPUT
+          echo "plugin_kind=${plugin_kind}" >> $GITHUB_OUTPUT
+          echo "plugin_kind_plural=${plugin_kind_plural}" >> $GITHUB_OUTPUT
+          echo "plugin_version=${plugin_version}" >> $GITHUB_OUTPUT
+          echo "plugin_dir=." >> $GITHUB_OUTPUT
+      # Fail if not a valid SemVer string
+      - name: Parse semver string
+        uses: booxmedialtd/ws-action-parse-semver@7784200024d6b3fc01253e617ec0168daf603de3
+        id: semver_parser
+        with:
+          input_string: ${{steps.split.outputs.plugin_version}}
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - uses: actions/github-script@v6
+        id: set-result
+        env:
+          PLUGIN_DIR: ${{steps.split.outputs.plugin_dir}}
+        with:
+          script: |
+            const fs = require('fs').promises;
+            const path = require('path');
+            const pluginFiles = await fs.readdir(process.env.PLUGIN_DIR);
+            if (pluginFiles.includes('Dockerfile')) {
+              return 'docker';
+            }
+            if (pluginFiles.includes('.goreleaser.yaml')) {
+              return 'go';
+            }
+          result-encoding: string
+  publish-plugin-to-hub:
+    timeout-minutes: 60
+    runs-on: large-ubuntu-monorepo
+    needs: prepare
+    if: needs.prepare.outputs.plugin_releaser == 'go'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: actions/cache@v3
+        with:
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+          key: ${{ runner.os }}-go-1.21.1-publish-plugin-to-hub-cache
+          restore-keys: |
+            ${{ runner.os }}-go-1.21.1-publish-plugin-to-hub-cache
+
+      - name: Set up Go 1.x
+        uses: actions/setup-go@v4
+        with:
+          go-version-file: ${{ needs.prepare.outputs.plugin_dir }}/go.mod
+          cache: false
+
+      - name: Use Node.js LTS
+        uses: actions/setup-node@v3
+        with:
+          node-version: 'lts/*'
+
+      - name: Install shell escape
+        run: |
+          npm install shell-escape@0.2.0
+
+      - name: Get Release Notes
+        id: release-notes
+        uses: actions/github-script@v6
+        with:
+          result-encoding: string
+          script: |
+            const shellescape = require('shell-escape');
+            const { data } = await github.rest.repos.getReleaseByTag({
```

Comment:
I don't think this will work unless they use release-please, also it uses the wrong owner name